src/myrtlespeech/model/hard_lstm.py

```diff
@@ -0,0 +1,561 @@
+"""This implementation of a hard LSTM can be exported to ONNX and run on
+ONNX Runtime (ORT). In order to capture the control flow (e.g. looping over
+timesteps), it is necessary to :py:meth`torch.jit.script` the module before
+:py:meth`torch.onnx.export`-ing it. In other words, the :py:class:`HardLSTM`
+below must be able to pass through the following pipeline:
+    ``nn.Module -> TorchScript -> ONNX -> ORT session``
+
+In order to sastify this constraint, our :py:class:`HardLSTM` class has a
```

Comment:
Nit: Generally the documentation has steered away from using pronouns. In this case `our` could be replaced with `the`?